src/components/Slide/index.jsx

```diff
@@ -2,48 +2,91 @@ import React from 'react';
 import {
   Card
 } from '@material-ui/core';
-import styled from 'styled-components';
-import { DragSource } from 'react-dnd';
+import { useDrag, useDrop } from 'react-dnd'
+import { connect } from 'react-redux'
+import { 
+  setActiveSlide,  
+  changeSlideOrder,
+} from '../../actions'
```

Comment:
Just a formatting thing (not a big deal) but referencing 'actions' here will work, no need to do long relative path searches like `../../../somefolder`